modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -86,24 +93,87 @@ In earlier versions of {product} When the Role-Based Access Control (RBAC) plugi
 
 This update ensures that the RBAC tab is hidden when the RBAC plugin is disabled.
 
-With this fix, the RBAC tab is no longer visible when the RBAC plugin is disabled, resulting in a cleaner and more intuitive user interface. link:{LinkRHIDPIssue}RHIDP-1449[(RHIDP-1449)]
+With this fix, the RBAC tab is no longer visible when the RBAC plugin is disabled, resulting in a cleaner and more intuitive user interface. 
+//link:{LinkRHIDPIssue}RHIDP-1449[RHIDP-1449]
 --
 
+Show configure access button for previously created simple permission policies in edit form::
++
+--
+In previous versions, when a user creates simple permission policies and later returns to edit the role, the *Configure Access* button does not appear if they wish to add conditional permission policies.
```

Comment:
```suggestion
In previous versions, when a user creates simple permission policies and later returns to edit the role, the *Configure Access* button does not appear if they want to add conditional permission policies.
```

Reply:
Yes to _want_ instead of _wish_ as a general rule. The other option is to end the sentence after _appear_, especially since the next sentence states that this issue affects being able to add conditional permission policies.